spec/ops/config_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'spec_helper'
+
+RSpec.describe Ops::Config do # rubocop:disable Lint/EmptyBlock
+end
```

Comment:
Why was this file added?

Reply:
because the file was blank and rubocop complained. I tried to disable the check and then re-enable it but it still complained so I added an empty block. I would haved added tests here if I knew what this code was supposed to be doing but I don't really know why the original authors decided to use a `hash` with `method_missing` if all possible options were known.

Reply:
I didn't realize it was an existing empty file. Strange that it wasn't populated but you aren't making it worse.